packages/mds-geography-service/repository/index.ts

```diff
@@ -58,17 +58,38 @@ class GeographyReadWriteRepository extends ReadWriteRepository {
 
   protected findGeographies = async (
     where: FindManyOptions<GeographyEntity>['where'],
```

Comment:
Why not use the `where` argument?

Reply:
Seemingly because it needs to work even if `includeMetadata` is false?

Reply:
That was the idea. I'll see if it is possible to generate the equivalent of that query just by using the "where" argument.

Reply:
I couldn't find a way to make the join and query on the json field with just the where clause. @nessur Is this possible? Still getting used to typeorm.